sourcegraph/sourcegraph
# Rohan/desktop404reporting

How to test:
- Run Sourcegraph Desktop (ensure pointed at this local server)
- add `view.openDevTools()` to `dom-ready` in `DefInfoView.js` in app
- Trigger error
- Ensure error is sent in `EventLogger.js` (sourcegraph/sourcegraph)
